<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordBox</class>
 <widget class="QWidget" name="PasswordBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>122</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLCDNumber" name="display">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>54</height>
      </size>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QToolButton" name="btn_3">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/red/3.png</normaloff>:/red/3.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="btn_1">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/red/1.png</normaloff>:/red/1.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="btn_8">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/red/8.png</normaloff>:/red/8.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="btn_2">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/red/2.png</normaloff>:/red/2.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="btn_4">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/red/4.png</normaloff>:/red/4.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="btn_6">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/red/6.png</normaloff>:/red/6.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="btn_7">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/red/7.png</normaloff>:/red/7.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="btn_5">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/red/5.png</normaloff>:/red/5.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="btn_9">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/red/9.png</normaloff>:/red/9.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QToolButton" name="btn_0">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/red/0.png</normaloff>:/red/0.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="btn_bs">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/plus_minus/clear.png</normaloff>:/plus_minus/clear.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="btn_run">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_0</tabstop>
  <tabstop>btn_1</tabstop>
  <tabstop>btn_2</tabstop>
  <tabstop>btn_3</tabstop>
  <tabstop>btn_4</tabstop>
  <tabstop>btn_5</tabstop>
  <tabstop>btn_6</tabstop>
  <tabstop>btn_7</tabstop>
  <tabstop>btn_8</tabstop>
  <tabstop>btn_9</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../../lib2/icons/digits/digits.qrc"/>
 </resources>
 <connections/>
</ui>
